snuba/cli/devserver.py

```diff
@@ -337,7 +337,7 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                             "subscriptions-scheduler",
                             "--entity=generic_metrics_distributions",
                             "--consumer-group=snuba-generic-metrics-distributions-subscriptions-schedulers",
-                            "--followed-consumer-group=snuba-generic-metrics-distributions-consumers",
+                            "--followed-consumer-group=snuba-gen-metrics-distributions-consumers",
```

Comment:
For example, https://github.com/getsentry/snuba/pull/6431/files#diff-27ffd2f11e990878aec700a2b0460871300553cc59992f0403c6782bdaf0763aR269